### PR TITLE
Tests for book 

### PR DESCRIPTION
**Pull request by nhanaa**

### CODE REVIEW

File: hello-spring/src/main/resources/openapi/paths/book-id.yaml [hunk 1]
```diff
@@ -10,7 +10,11 @@ delete:
       description: UUID of the book to delete
   responses:
     "200":
-      description: Successfully delete the selected book
+      description: The book deleted with its UUID
```
Comment (ncovercash) [inline hello-spring/src/main/resources/openapi/paths/book-id.yaml]: Any particular reason you changed this to respond with the deleted entity?

Comment (nhanaa) [reply hello-spring/src/main/resources/openapi/paths/book-id.yaml]: I just want to to be able to write test to make sure the deleted entity is correct but I guess it is not really necessary.

Comment (ncovercash) [reply hello-spring/src/main/resources/openapi/paths/book-id.yaml]: Gotcha, was just curious.  I'd recommend having it return a 204/no content, mainly for clarity.  For testing, you can do the DELETE then check that GET returns a 404

Comment (nhanaa) [reply hello-spring/src/main/resources/openapi/paths/book-id.yaml]: So should I change the DELETE request to return 204 status code instead of 200?

Comment (ncovercash) [reply hello-spring/src/main/resources/openapi/paths/book-id.yaml]: Yes; 204 is a special status code that indicates OK but ensures the client expects no response/void

Comment (ncovercash) [reply hello-spring/src/main/resources/openapi/paths/book-id.yaml]: On that note post should be a 201 (created), too. Shouldn't need to change the Java side for that though

**Review by ncovercash — APPROVED** [review]

**Pull request merged**